app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const serverless = require('serverless-http');
+const bodyParser = require('body-parser');
+const sequelize = require('./config');
+require('dotenv').config();
```

Comment:
Do we need `dotenv`?

I believe that for `genezio local` we are automatically loading env with `dotenv` for the user and for `genezio deploy` we don't need to load them because they are already exposed in `process.env`.

Can you test and this is not require please remove `dotenv` from the dependency list.